yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1467,7 +1470,14 @@ func processCollectedSql(fpath string, stmt string, formattedStmt string, objTyp
 	//update about sqlStmt in the summary variable for the report generation part
 	createObjStmt := createObjRegex.FindStringSubmatch(formattedStmt)
 	if createObjStmt != nil {
-		objName = createObjStmt[objNameIndex]
+		if slices.Contains([]string{"INDEX", "POLICY", "TRIGGER"}, objType) {
```

Comment:
Could there be other types that could require same treatment? Just do a scan once ..

Reply:
I don't think there can be anything other than these.